### PR TITLE
Update charter.md

### PR DESCRIPTION
**Pull request by debcooley**
go easy, this might have been my first PR (stop laughing!).

I added 'M' to the name of the working group.  
I took a stab at making the goals less 'ordered'.

Please feel free to comment, correct or whatever (just don't laugh).

Deb

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -20,20 +20,20 @@ previous Secure Shell WG has been closed and its documents published. In the
...
+chartered to generally maintain the SSH protocol documentation. 
+
+The initial goals of this new working group are:   
+
+* to update the RFCs documenting SSH to reflect what is implemented and deployed in practice.
+  In particular, the working group will document the OpenSSH certificate structure, the SSH agent
+  protocol, and SFTP, as they are commonly implemented.
+
+* to update and maintain the list of cryptographic algorithms used by SSH. This includes documenting
+  existing algorithms, deprecating unsafe algorithms, defining new algorithms (such as post-quantum),
```
Comment (job) [inline charter.md]: Perhaps “selecting” instead of “defining”?

**Comment by job** [comment]
Looks great

File: charter.md (hunk 1 above)
Comment (debcooley) [reply charter.md]: yes, that is better.  TY

Comment (job) [reply charter.md]: If you change the word and commit again, and push again, it should show up in this PR

Comment (debcooley) [reply charter.md]: Ha!  I think I did it.  Thanks for the push.

File: charter.md [hunk 2]
```diff
@@ -45,6 +45,6 @@ implementation and deployment. Protocol documents should not be submitted to
 the IESG for publication before they have at least two demonstrably
 interoperable implementations.
 
-Defining new certificate types or trust mechanisms is out of scope. New
-transports for SSH are out of scope. While defining how SSH uses cryptographic
+Out of scope initially include:  defining new certificate types or trust mechanisms; new
```
Comment (DavidSchinazi) [inline charter.md]: Can we make this a list too?

**Review by DavidSchinazi — COMMENTED** [review]
Looks great!

File: charter.md (hunk 2 above)
Comment (debcooley) [reply charter.md]: done!

**Review by DavidSchinazi — APPROVED** [review]

**Pull request merged**